coco/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from .validators import validate_file_extension
+
+
+def validate_file_extension(value):
```

Comment:
this could be done on js, with alert message and rest of the event can be stopped. No need to take it to backend